client/src/Utils/ControllerHooks/useExposuresSaving.ts

```diff
@@ -46,6 +44,12 @@ const useExposuresSaving = (exposuresAndFlightsVariables: ExposureAndFlightsDeta
     const saveExposureAndFlightData = async () : Promise<void> => {
         let { exposures, wereFlights, wereConfirmedExposures, exposuresToDelete } = exposuresAndFlightsVariables.exposureAndFlightsData;
         let filteredExposures : (Exposure | DBExposure)[] = [];
+        const saveExposureAndFlightDataLogger = logger.setup({
+            workflow: 'Saving Exposures And Flights',
+            service: Service.CLIENT,
```

Comment:
You don't need to pass it service. it does it automatically

Reply:
delete it from all the setups

Reply:
so why is he an optiontal parameter in setup?

Reply:
It probably doesn't need to be there

Reply:
fixed